### PR TITLE
Add instructions how to use it with npm

### PR DESCRIPTION
**Pull request by marcindobry**
Some people still like npm, this updates the readme to give people options :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ See the official [prepack.io](http://prepack.io) website for an introduction and
 
 ## How to use Prepack
 
-Install the CLI via yarn,
+Install the CLI via npm,
 
 ```bash
+$ npm install -g yarn
 $ yarn global add prepack
```
Comment (NTillmann) [inline README.md]: Please also remove `$ yarn global add prepack` here, as it doesn't make sense.

**Comment by marcindobry** [comment]
It was supposed to be a joke since the README has two options of installing prepack, both using yarn :/

File: README.md (hunk 1 above)
Comment (marcindobry) [inline README.md]: It was supposed to be a joke since the README has two options of installing prepack, both using yarn :/

**Comment by NTillmann** [comment]
Superseded by #786.

**Pull request closed without merging**